var/spack/repos/builtin/packages/paraver/package.py

```diff
@@ -7,24 +7,61 @@
 import os
 
 
-class Paraver(Package):
+class Paraver(AutotoolsPackage):
     """"A very powerful performance visualization and analysis tool
         based on traces that can be used to analyse any information that
         is expressed on its input trace format.  Traces for parallel MPI,
         OpenMP and other programs can be genereated with Extrae."""
```

Comment:
Can you add a note saying that this package is deprecated and users should use wxparaver instead?

Reply:
Done